packages/core/src/etherscan.ts

```diff
@@ -259,14 +259,6 @@ export const attemptVerification = async (
   }
 }
 
-export const isSupportedNetworkOnEtherscan = (chainId: number): boolean => {
-  const chainConfig = [...customChains, ...builtinChains].find(
-    (config) => config.chainId === chainId
-  )
-
-  return chainConfig !== undefined
-}
```

Comment:
We had two copies of this function, so I deleted this in favor of just using `isVerificationSupportedForNetwork`.